### PR TITLE
remove copyright notice

### LINKED ISSUE #141: Remove Scipp-specifics, or make them optional?
If we want to make this more widely usable, we should consider removing as many Scipp-specifics as possible, or least make them optional.

Start by addressing the simple and obvious, it does not have to be perfect in the first iteration!

Here is a list (maybe incomplete) of potential offenders:
```
$ grep -r scipp template/
template/CODE_OF_CONDUCT.md:scipp[at]ess.eu.
template/requirements/make_base.py:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/requirements/make_base.py:        org = "scipp"
template/requirements/make_base.py:    if repo == "scipp":
template/requirements/make_base.py:        base = "https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly"
template/docs/about/index.md.jinja:Older versions of the documentation pages can be found under the assets of each [release](https://github.com/scipp/{{projectname}}/releases).
template/docs/about/index.md.jinja:{{prettyname}} is hosted and developed [on GitHub](https://github.com/scipp/{{projectname}}).
template/docs/conf.py.jinja:    'scipp': ('https://scipp.github.io/', None),
template/docs/conf.py.jinja:    'scipp._scipp.core.DataArray': 'scipp.DataArray',
template/docs/conf.py.jinja:    'scipp._scipp.core.Dataset': 'scipp.Dataset',
template/docs/conf.py.jinja:    'scipp._scipp.core.DType': 'scipp.DType',
template/docs/conf.py.jinja:    'scipp._scipp.core.Unit': 'scipp.Unit',
template/docs/conf.py.jinja:    'scipp._scipp.core.Variable': 'scipp.Variable',
template/docs/conf.py.jinja:    'scipp.core.data_group.DataGroup': 'scipp.DataGroup',
template/docs/conf.py.jinja:        {"name": "{{ link }}", "url": "https://scipp.github.io{% if link == 'Scipp' %}{% else %}/{{ link|lower }}{% endif %}"},
template/docs/conf.py.jinja:    "--Session.metadata=scipp_sphinx_build=True",
template/docs/conf.py.jinja:    import scipp as sc
template/docs/conf.py.jinja:# Using normalize whitespace because many __str__ functions in scipp produce
template/docs/_templates/doc_version.html:Current {{ project }} version: {{ version }} (<a href="https://github.com/scipp/{{ project|lower }}/releases">older versions</a>).
template/pyproject.toml.jinja:"Documentation" = "https://{{orgname}}.github.io{% if projectname != 'scipp' %}/{{projectname}}{% endif %}"
template/.github/dependabot.yml:  # noise in all template instances. Instead dependabot.yml in scipp/copier_template
template/.github/dependabot.yml:      - dependency-name: "scipp"
template/.github/workflows/ci.yml:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/.github/workflows/unpinned.yml:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/.github/workflows/test.yml:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/.github/workflows/nightly_at_main.yml:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/.github/workflows/docs.yml:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/.github/workflows/nightly_at_release.yml:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/.github/workflows/release.yml:# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
template/.github/workflows/release.yml:      - run: conda mambabuild --channel conda-forge --channel scipp --no-anaconda-upload --override-channels --output-folder conda/package conda
template/.github/workflows/release.yml:      - run: anaconda --token ${{ secrets.ANACONDATOKEN }} upload --user scipp --label main $(ls conda-package-noarch/*.tar.bz2)
```

### PR DESCRIPTION
**Pull request by jokasimr**
Fixes #141

### CODE REVIEW

File: template/docs/conf.py.jinja [hunk 1]
```diff
@@ -59,8 +61,8 @@ autodoc_type_aliases = {
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
-    'numpy': ('https://numpy.org/doc/stable/', None),
-    'scipp': ('https://scipp.github.io/', None),
+    'numpy': ('https://numpy.org/doc/stable/', None),{% if orgname == 'scipp'  %}
+    'scipp': ('https://scipp.github.io/', None),{% endif %}
```
Comment (YooSunYoung) [inline template/docs/conf.py.jinja:65]: How about this...?

```suggestion
    'numpy': ('https://numpy.org/doc/stable/', None),
    '{{orgname}}': ('https://{{orgname}}.github.io/', None),
```

Comment (SimonHeybrock) [reply template/docs/conf.py.jinja:65]: Don't see why? What are the odds that someone has org-wide github pages and wants to use this template?

Comment (YooSunYoung) [reply template/docs/conf.py.jinja:65]: Then what's the reason of having this if condition here...?

Comment (SimonHeybrock) [reply template/docs/conf.py.jinja:65]: Because Scipp has it, and (almost) all other projects in the org use it?

Comment (YooSunYoung) [reply template/docs/conf.py.jinja:65]: Apart from this, it looks good to me...!

**Review by YooSunYoung — APPROVED** [review]

**Pull request merged**